android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapFragment.java

```diff
@@ -424,9 +423,10 @@ void setHighBandwidthMap(boolean isHighBandwidth) {
         final ClientPrefs.MapTileResolutionOptions tileType = prefs.getMapTileResolutionType();
         final int idxTileType = tileType.ordinal();
         if (idxTileType > 0) {
+            hasNetwork = true; // always update the map type
+
             if (tileType == ClientPrefs.MapTileResolutionOptions.NoMap) {
-                mTextViewIsLowResMap.setVisibility(View.VISIBLE);
-                mTextViewIsLowResMap.setText(getActivity().getString(R.string.map_turned_off));
```

Comment:
again, old code i realize, but mTextViewIsLowResMap is now more aptly named
mTextViewMapResolutionInfo